### PR TITLE
docfix for minimize in TrainResult

### PR DESCRIPTION
**Pull request by ananyahjha93**

### CODE REVIEW

**Comment by s-rog** [comment]
thanks for the quick reply on slack, I was also thinking that maybe it can be renamed to something more intuitive like backprop

**Review by Borda — APPROVED** [review]

**Comment by Borda** [comment]
test is failing on master - https://github.com/PyTorchLightning/pytorch-lightning/commit/be0438bb47596447b9485f0bf507b474660f0f00#commitcomment-41807345

**Review by SkafteNicki — APPROVED** [review]

**Comment by ananyahjha93** [comment]
@s-rog I don't think backprop is more intuitive in this case. Minimize makes more sense in way that you want to pass it a loss tensor that you wish to minimize.
